resources/views/errors/500.blade.php

```diff
@@ -1,28 +1,5 @@
-<html>
-<head>
-    <title>
-        @lang("error.back_soon")
-    </title>
-    <style>
-        body {
-            background-color: #f9f9f9;
-            color: #333;
-            text-align: center;
-            text-shadow: 0 1px 0 #fff;
-            font-size: 1.8em;
-        }
-        .missing {
-            width: 250px;
-            margin: 0 auto;
-            margin-top: 50px;
-            padding: 40px;
-        }
-    </style>
-</head>
-<body>
-<div class="missing">
-    <h2>@lang("error.back_soon")</h2>
-    @lang("error.back_soon_description")
-</div>
-</body>
-</html>
\ No newline at end of file
+@extends('errors::minimal')
+
+@section('title', @lang("error.back_soon"))
```

Comment:
I'm also getting an error here: 

`[2020-06-30 21:50:30] production.ERROR: Call to undefined function lang() (View: /usr/share/nginx/html/resources/views/errors/500.blade.php) {"userId":2,"exception":"[object] (Facade\\Ignition\\Exceptions\\ViewException(code: 0): Call to undefined function lang() (View: /usr/share/nginx/html/resources/views/errors/500.blade.php) at /usr/share/nginx/html/resources/views/errors/500.blade.php:3)`

Reply:
Thanks I will fix this

Reply:
[Fix](https://github.com/Attendize/Attendize/pull/803/commits/dadfd5f21e872d13d88375cfcf421c12c8946de6)